data/json/recipes/tools/containers.json

```diff
@@ -1149,5 +1149,44 @@
     "proficiencies": [ { "proficiency": "prof_metalworking" } ],
     "qualities": [ { "id": "CHISEL", "level": 3 } ],
     "components": [ [ [ "scrap_aluminum", 120 ], [ "material_aluminium_ingot", 10 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
```

Comment:
Shouldn't it be light execise to glue the glass sheets together?

Reply:
This recipe also includes cutting the glass sheets into appropriate sizes, though, hence it being Brisk. Having said that, that may be overkill for glass cutting. I have not done it myself so I am more than open to editing that accordingly.

Reply:
https://youtu.be/gfmRLTxVFmg?t=103

Unless fish tanks get some bulletproof glass instead of the sheets you buy in home improvement stores that doesn't look similar to blacksmithing levels of exertion

Reply:
You are right, I'll change this to light exercise.